charts/sk8s/templates/prometheus-configmap.yaml

```diff
@@ -29,6 +29,6 @@ data:
       # scheme defaults to 'http'.
       metrics_path: '/prometheus'
       static_configs:
-      - targets: ['{{ template "fullname" . }}-topic-gateway:{{ .Values.topicGateway.service.externalPort }}','{{ template "fullname" . }}-event-dispatcher:{{ .Values.eventDispatcher.service.externalPort }}']
+      - targets: ['{{ template "fullname" . }}-topic-gateway:{{ .Values.topicGateway.service.externalPort }}','{{ template "fullname" . }}-functionController:{{ .Values.functionController.service.externalPort }}']
```

Comment:
should this be `- targets: ['{{ template "fullname" . }}-topic-gateway:{{ .Values.topicGateway.service.externalPort }}','{{ template "fullname" . }}-function-controller:{{ .Values.functionController.service.externalPort }}']`

Reply:
good catch